components/cli/vendor/github.com/docker/docker/api/types/swarm/runtime/gen.go

```diff
@@ -0,0 +1,3 @@
+//go:generate protoc -I . --gogofast_out=import_path=github.com/docker/docker/api/types/swarm/runtime:. plugin.proto
```

Comment:
Brought in by this PR:
* https://github.com/moby/moby/pull/33575 Plugins on swarm

as well as the following few diffs